### PR TITLE
Second Annual Workshop

### PR DESCRIPTION
**Pull request by odow**
@mlubin can I get push access please?

### CODE REVIEW

File: developersmeetup_2018/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Location
+
+The workshop will be held at the Institut de Mathématiques de Bordeaux, University of Bordeaux on the Thursday and Friday of the week prior to ISMP 2018 (i.e. June 28-29).
+
+A variety of information about Bordeaux, as well as how to get there, and possible accommodation options, is available on the [ISMP conference website](https://ismp2018.sciencesconf.org).
+
+## Call for participants
+
+The workshop is open for anybody to attend, however it is aimed at JuMP developers rather than users. In particular, we encourage new contributors and those who have not met the core development team to attend.
```
Comment (mlubin) [inline developersmeetup_2018/index.md]: s/JuMP developers/developers of mathematical optimization software who use JuMP/

File: developersmeetup_2018/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Participants are invited to give a 20 minute talk about how they are using JuMP. Talks should be aimed at a technical audience and demonstrate what you are using JuMP for, as well as the features that you like/dislike/have on your wishlist. Depending on the number of participants, we may not be able to accommodate all those who wish to talk. ([Here](https://www.youtube.com/watch?v=esOe5saQRKY&list=PLzK_rUGmc3o6EwPOCUCvBAbMJeYBS8PyY) is a playlist of talks from the previous workshop.)
```
Comment (mlubin) [inline developersmeetup_2018/index.md]: s/about how they are using JuMP/about their work that is related to JuMP/

**Comment by mlubin** [comment]
Done. Could we tidy up the directory structure?
`developersmeetup/` goes to `meetings/mit2017` and this year's goes to `meetings/bordeaux2018`? We can leave a redirect at `developersmeetup/`

**Comment by odow** [comment]
Followed the redirect instructions from https://help.github.com/articles/redirects-on-github-pages/ but didn't test locally.

File: meetings/bordeaux2018/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+layout: common
+---
+
+# The Second Annual JuMP Developers Workshop
+
+June 28-29, 2018, at the Institut de Mathématiques de Bordeaux, University of Bordeaux
```
Comment (blegat) [inline meetings/bordeaux2018/index.md]: Missing an ending dot :-P

File: meetings/bordeaux2018/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+The workshop will be held at the Institut de Mathématiques de Bordeaux, University of Bordeaux on the Thursday and Friday of the week prior to ISMP 2018 (i.e. June 28-29).
+
+A variety of information about Bordeaux, as well as how to get there, and possible accommodation options, is available on the [ISMP conference website](https://ismp2018.sciencesconf.org).
+
+## Call for participants
+
+The workshop is open for anybody to attend, however it is aimed at developers of mathematical optimization software who use JuMP rather than users. In particular, we encourage new contributors and those who have not met the core development team to attend.
+
+Participants are invited to give a 20 minute talk about their work that is related to JuMP. Talks should be aimed at a technical audience and demonstrate what you are using JuMP for, as well as the features that you like/dislike/have on your wishlist. Depending on the number of participants, we may not be able to accommodate all those who wish to talk. ([Here](https://www.youtube.com/watch?v=esOe5saQRKY&list=PLzK_rUGmc3o6EwPOCUCvBAbMJeYBS8PyY) is a playlist of talks from the previous workshop.)
```
Comment (blegat) [inline meetings/bordeaux2018/index.md]: "20 minutes" or "20-minute" I think

**Comment by mlubin** [comment]
@joaquimg, you may want to follow this repo

File: meetings/bordeaux2018/index.md (hunk 4 above)
Comment (mlubin) [reply meetings/bordeaux2018/index.md]: "20-minute"

**Comment by mlubin** [comment]
@odow, could you test the redirect locally? It's going to be useful anyway for you to be able to preview the site locally in the future.

File: meetings/bordeaux2018/index.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Please contact o.dowson at auckland.ac.nz if you would like to attend.
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: Maybe `jump-dev-committee@googlegroups.com`? I've allowed moderated posting from non-members, and set you as a moderator :)

File: meetings/bordeaux2018/index.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+---
+layout: common
+---
+
+# The Second Annual JuMP Developers Workshop
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: Do we want to brand it as "JuMP-dev" for short?

File: meetings/bordeaux2018/index.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+### Friday June 29
+
+ - Morning: Participant talks
+ - Afternoon: Developer collaboration
+
+### Sunday July 1
+
+ - ISMP 2018 begins.
+
+## Contact
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: We should list the committee here.

**Comment by odow** [comment]
Okay so that was a massive hassle to get everything installed and working on Windows. I haven't pushed the latest `Gemfile.lock` as I'm not sure if it's a good thing to do since I had to follow about 10 different stack-overflow instructions and I had to remove some things (like `group: :jekyll_plugins` from the Gemfile) in order to get it to run locally. 

I also renamed `redirect.html` to `redirect.html.old` since it was conflicting with the redirection. For some reason it tries to redirect to `docs.julialang.org/en/latest` so it may just be a relic and can be removed?

File: _layouts/redirect.html.old [hunk 8]
```diff
@@ -1,3 +1,6 @@
+<!--
```
Comment (mlubin) [inline _layouts/redirect.html.old]: What is this file doing? Can we remove it?

File: meetings/mit2017/index.md [hunk 9]
```diff
@@ -1,27 +1,28 @@
 ---
 layout: common
+redirect_from: "/developersmeetup/"
 ---
 
-# JuMP Developers Meetup/Workshop
+# First Annual JuMP-dev Workshop
```
Comment (mlubin) [inline meetings/mit2017/index.md]: I don't know if we should change the name retroactively, or if we do at least leave a note of the original name.

File: _layouts/redirect.html.old (hunk 8 above)
Comment (odow) [reply _layouts/redirect.html.old]: If I left it, the jekyll redirection didnt' work, and `/developersmeetup/` redirected to `docs.julialang.org/en/latest/developersmeetup`

File: index.md [hunk 10]
```diff
@@ -11,7 +15,7 @@ title:  JuliaOpt&#58; Optimization packages for the Julia language
 The [JuliaOpt GitHub organization](https://github.com/JuliaOpt) is home to a number of optimization-related packages written in Julia. Its purpose is to facilitate collaboration among developers of a tightly integrated set of packages for mathematical optimization.
 
 - **Overview**: [presentation](https://www.youtube.com/watch?v=7LNeR299q88) and [workshop](https://www.youtube.com/watch?v=nnL7yLMVu6c) from JuliaCon 2015
-- **Meetings**: [developers meetup 2017](/developersmeetup/index.md)
+- **Developer Workshops**: [2017](meetings/mit2017/), [2018](/meetings/bordeaux2018/)
```
Comment (mlubin) [inline index.md]: I'm okay merging and iterating on the workshop page if we hold off on linking to it publicly for a bit.

File: index.md [hunk 11]
```diff
@@ -3,6 +3,10 @@ layout: default
 title:  JuliaOpt&#58; Optimization packages for the Julia language
 ---
 
+# JuMP Developers Workshop 2018, Bordeaux
```
Comment (mlubin) [inline index.md]: Call it JuMP-dev here also?

**Comment by odow** [comment]
Seems like everything built fine. The re-direction is working.

**Pull request merged**